env/file_system_tracer.cc

```diff
@@ -0,0 +1,306 @@
+// Copyright (c) 2019-present, Facebook, Inc.  All rights reserved.
+// This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+//
+
+#include "env/file_system_tracer.h"
+
+namespace ROCKSDB_NAMESPACE {
+
+namespace {
+using namespace std::chrono;
+uint64_t Timestamp() {
+  return duration_cast<microseconds>(system_clock::now().time_since_epoch())
+      .count();
+}
+}  // namespace
+
+IOStatus FileSystemTracingWrapper::NewWritableFile(
+    const std::string& fname, const FileOptions& file_opts,
+    std::unique_ptr<FSWritableFile>* result, IODebugContext* dbg) {
+  uint64_t start_time = Timestamp();
```

Comment:
For consistency sake, we can use ```StopWatch```, like here https://github.com/facebook/rocksdb/blob/master/db/db_impl/db_impl_write.cc#L1404. What do you think?

Reply:
```StopWatch``` requires Env pointer and in ```file_system_tracer.cc``` I don't have Env. 
It also needs statistics pointer for ```StopWatch```.

I can use ```StopWatchNano``` also as it requires ```Env``` not ```Statistics```.  Should I create Env object with ```Env::Default()``` in ```file_system_tracer.cc```?

Reply:
Yes, ```StopWatchNano``` with ```Env::Default()``` should be okay. I'm not sure if using ```StopWatch/StopWatchNano``` will make it more portable, but rest of the codebase uses those, so I think its better to be consistent.